servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -4,15 +4,16 @@ The 3scale Istio Adapter allows you to label a service running within the {Produ
 Prerequisites:
 
 * {ProductName} 0.7.0+
-* A working 3scale account (https://www.3scale.net/signup/[SaaS] or https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/infrastructure/onpremises-installation[On-Premises])
+* A working 3scale account (https://www.3scale.net/signup/[SaaS] or https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/infrastructure/onpremises-installation[3scale 2.5 On-Premises])
```

Comment:
The text for the link has been changed, however the URL is still going to 2.4. For 3scale 2.5, the structure has changed, so it's not as simple as just updating the number in the URL.

**Correct link:** https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/onpremises-installation

Reply:
I've updated this link.